config/projects.yml

```diff
@@ -269,12 +269,12 @@ intl-bundle:
       php: ['7.1']
       versions:
         symfony: ['2.8', '3.1', '3.2']
-        sonata_user: ['2', '3']
+        sonata_user: ['3']
     2.x:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
       versions:
         symfony: ['2.3', '2.7', '2.8', '3.1', '3.2']
-        sonata_user: ['2', '3']
```

Comment:
Maybe we should leave this unchanged, just to not break people using 2.x of SonataUser with 2.x of IntBundle?

IMO if we can't change composer.json (this line: https://github.com/sonata-project/SonataIntlBundle/blob/2.x/composer.json#L37) there is no reason to change this.

It doesn't matter a lot because it is only used here: https://github.com/sonata-project/SonataIntlBundle/blob/2.x/Timezone/UserBasedTimezoneDetector.php

So, if it is on the same namespace and it has `getTimezone()` function, it will "work", or atleast not break on any version.

WDYT @greg0ire ?